### ISSUE TITLE
Laravel 5.3

### ISSUE THREAD

**Issue by mstaack** [yajra/laravel-oci8#199]
@yajra added basic laravel 5.3 support
https://github.com/yajra/laravel-oci8/commit/cd9a4bcd17b076ffb12133062799879584f99a05

**Comment by mstaack** [comment]
looks like there is something wrong
```
                                                                                                                                                                         
  [Illuminate\Database\QueryException]                                                                                                                                   
  oci_bind_by_name(): ORA-01036: illegal variable name/number (SQL: select * from all_tables where upper(owner) = upper(system) and upper(table_name) = upper(migration  
  s))                                                                                                                                                                    
                                                                                                                                                                         

                                                               
  [ErrorException]                                             
  oci_bind_by_name(): ORA-01036: illegal variable name/number
```

**Comment by yajra** [comment]
@mstaack thanks for the initiative on 5.3 support. Will try to check it too later!

**Comment by yajra** [comment]
I just looked at this and it seems like a lot of breaking changes occurs. Need to re-work on `pdo-via-oci8` package as most of the functions fails there. Not sure though why.. Weird error `oci_bind_by_name(): ORA-01036: illegal variable name/number`! 

Will have a look at it again this evening after work. ^_^

**Comment by mstaack** [comment]
yeah query builders return collections now...

**Comment by mstaack** [comment]
@yajra did you have time so far to look at it?

**Comment by yajra** [comment]
Not yet, need to do some work deadlines. Maybe this weekend when I had the chance.

**Comment by yajra** [comment]
@mstaack I just pushed a fix and I think the package is initially working on Laravel 5.3 now. I just tested on migration but haven't check the full functionality yet.

**Comment by mstaack** [comment]
will give it a try...thanks

**Comment by wffranco** [comment]
Don't work in 5.3 ... In case this help you, I created a totally new project and received that message:

Your requirements could not be resolved to an installable set of packages.                                                                                           
  Problem 1                                                                                                                                                          
    - The requested package yajra/laravel-oci8 5.3.* is satisfiable by yajra/laravel-oci8[5.3.x-dev] but these conflict with your requirements or minimum-stability.

----

Is a full new project, not installed anything else.

Thanks for your work.

**Comment by yajra** [comment]
@wffranco you got that issue because we don't have a stable release yet and I assume that you are now using the newly released 5.3 stable version. That issue should be be fixed when I tag this branch later. Thanks!

**Comment by yajra** [comment]
Guys, I just released v5.3.0, kindly check it out if it works! Thanks!

**Comment by wffranco** [comment]
@yajra that's exactly what I mean, installing it on laravel 5.3.0

Write the error just in case that help you in something.

Now was installed. Thanks a lot again for your work. It's time for testing :smile:
